src/tests/multihost/ad/test_dyndns.py

```diff
@@ -43,6 +43,25 @@ def enable_dns_forwarders():
     request.addfinalizer(enable_dns_forwarders)
 
 
+@pytest.fixture(scope="class")
+def reverse_zone(session_multihost, request):
+    """ Creates reverse zones """
+    dns = ADDNS(session_multihost.ad[0])
+    ip = str(session_multihost.ad[0].ip)
```

Comment:
Since you don't use ip anywhere except where it's split, you could change this to:

`ip = str(session_multihost.ad[0].ip).split(".")`

Then for network you could simplify this to:

`network = f'{ip[2]}.{ip[1]}.{ip[0]}.in-addr.arpa'`

Reply:
Thanks!